base_layer/core/src/validation/block_validators.rs

```diff
@@ -47,25 +46,28 @@ impl<B: BlockchainBackend> Validation<Block, B> for StatelessValidator {
     /// 1. Is the accounting correct?
     /// 1. Are all inputs allowed to be spend (Are the feature flags satisfied)
     fn validate(&self, block: &Block) -> Result<(), ValidationError> {
+        let rules = ConsensusManager::<B>::default();
```

Comment:
No, this is objectively bad.

I'm all for trying to remove the dep on `ConsensusManager` but this is not the way to do it.

A better way to do _that_ is to change `check_accounting_balance` to

```
fn check_accounting_balance(block: &Block, reward: MicroTari,  factories: &CryptoFactories) -> Result<(), ValidationError>
```

and then only store the emission curve in StateLessValidator.

OR we remove the accounting check from StatelessValidator
OR we move the emission curve instance into ConsensusConsts (and change the name to something more apropo)

Reply:
Need some 3rd opinions here